.claude/hooks/verify-fast.sh

```
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+# hook 전용 빠른 검증 (Stop 이벤트마다 실행)
+# 1~2초 안에 끝나는 grep 3개만 포함. ./gradlew test는 제외.
+# 전체 체크리스트(테스트 포함)는 수동 /verify 또는 subagent에서 실행.
+
+cd "$(git rev-parse --show-toplevel 2>/dev/null || echo .)"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)